test/e2e/storage/vsphere/vsphere_volume_vsan_policy.go

```diff
@@ -160,10 +156,8 @@ var _ = utils.SIGDescribe("Storage Policy Based Volume Provisioning [Feature:vsp
 		e2elog.Logf("Invoking test for VSAN storage capabilities: %+v", scParameters)
 		err := invokeInvalidPolicyTestNeg(client, namespace, scParameters)
 		framework.ExpectError(err)
-		errorMsg := "invalid option \\\"objectSpaceReserve\\\" for volume plugin kubernetes.io/vsphere-volume"
```

Comment:
seems like there was one more level of escaping here in that the error message searched for actually contains `\"objectSpaceReserve\"`

not sure if thats due to the behavior of `strings.Contains` or what